masterlist.yaml

```diff
@@ -5396,22 +5396,10 @@ plugins:
   - name: '3DNPC.esp'
     url:
       - link: 'http://3dnpc.com/2017/11/20/3dnpc-v3-4-for-oldrim-and-sse/'
-        name: 'Interesting NPCs'
+        name: 'Interesting NPCs SSE Beta'
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/29194/'
-        name: 'Interesting NPCs SE (3DNPC)'
-    tag:
-      - -C.Acoustic
-      - -C.Climate
-      - -C.Encounter
-      - -C.Location
-      - -C.Music
-      - -C.Name
-      - -C.RecordFlags
-      - -C.Regions
-      - -C.Water
-      - Keywords
-      - Invent.Add
-      - -Invent.Remove
+        name: 'Interesting NPCs'
+    inc: [ 'Open Cities Skyrim.esp' ]
```

Comment:
You'll probably still get complaints from anyone using the patch, despite the warnings.

Reply:
True, we could add a `detail` stating that the official patch is unsupported & not guaranteed to solve incompatibilities as stated by the author of the mod. Or do you think it'd be better to just hide the incompatibility message if they have the patch?

Reply:
Hiding is fine, though the first message could be added to the patch itself.
Something along the lines of this perhaps.

> This patch is unsupported by its author, use at your own risk as it is not guaranteed to solve incompatibilities.